src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
```

Comment:
It's a good practice to check if the element you are trying to select actually exists in the DOM. You can add a condition to check if the logo is not null or undefined before proceeding.